### PR TITLE
add explicit .npmignore because we want to include vega-lite

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by kanitw** [comment]
> add explicit .npmignore because we want to include vega-lite

you mean include __only__ `vega-lite*` ?

**Comment by domoritz** [comment]
To make this more clear. npm uses .gitignore if no .npmignore is present.

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,3 @@ vega-lite*.map
 spec.json
 npm-debug.log
 instance.json
-coverage
```
Comment (kanitw) [inline .gitignore]: Why not exclude coverage?

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+test/log
+test/log/difflist.json
+node_modules
+bower_components
+data
+editor
```
Comment (kanitw) [inline .npmignore]: I guess we’ll have to remove “editor” once we migrate.

File: .gitignore (hunk 1 above)
Comment (domoritz) [inline .gitignore]: I never have coverage locally. Reverted.

File: .npmignore (hunk 2 above)
Comment (domoritz) [inline .npmignore]: I don't think it hurts, though.

Comment (kanitw) [inline .npmignore]: Sure – just a note for cleanliness.

File: .gitignore (hunk 1 above)
Comment (kanitw) [inline .gitignore]: lol

**Pull request merged**